src/pages/RegisterAndLogin/Components/Inputs.tsx

```diff
@@ -42,7 +42,7 @@ const Inputs = ({ page }: { page: string }) => {
   const [open, setOpen] = React.useState(false);
   const toggleHandle = () => setOpen(!open);
 
-  const HandleOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+  const HandleEmailOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
handlers generally it starts with small letter
```suggestion
  const handleEmailOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```